x-pack/docs/en/security/authorization/privileges.asciidoc

```diff
@@ -201,15 +201,18 @@ Privilege to delete an index.
 Privilege to index and update documents. Also grants access to the update
 mapping action.
 
+`maintenance`::
+Privilege to refresh, flush, synced_flush, force merge
```

Comment:
```Suggestion
Permits refresh, flush, synced flush and force merge operations. No privilege to read or write index data or otherwise manage the index.
```